doxygen/doxygen.config.in

```diff
@@ -293,15 +293,15 @@ GENERATE_TAGFILE       =
 ALLEXTERNALS           = NO
 EXTERNAL_GROUPS        = YES
 EXTERNAL_PAGES         = YES
-PERL_PATH              = /usr/bin/perl
+#PERL_PATH              = /usr/bin/perl
```

Comment:
Why do we need to comment `PERL_PATH` out?

Reply:
Same as for the other comment.